pkg/apis/pravega/v1beta1/pravegacluster_types.go

```diff
@@ -1585,3 +1587,15 @@ func (p *PravegaCluster) NewApplicationEvent(name string, reason string, message
 	}
 	return &event
 }
+
+// OperatorName returns the operator name
```

Comment:
@AndreyAdnreyev  Are you planning to make changes in charts for the environment variable?

Reply:
This variable already exists in [charts](https://github.com/pravega/charts/blob/master/charts/pravega-operator/templates/operator.yaml). 
`        - name: OPERATOR_NAME
          value: {{ template "pravega-operator.fullname" . }}`